django/db/backends/base/schema.py

```diff
@@ -276,10 +276,15 @@ def table_sql(self, model):
                 if constraint
             ),
         }
-        if model._meta.db_tablespace:
-            tablespace_sql = self.connection.ops.tablespace_sql(
-                model._meta.db_tablespace
-            )
+        db_tablespace = None
+        if settings.DATABASES[self.connection.alias].get("DEFAULT_TABLESPACE"):
+            db_tablespace = settings.DATABASES[self.connection.alias][
+                "DEFAULT_TABLESPACE"
+            ]
+        elif model._meta.db_tablespace:
+            db_tablespace = model._meta.db_tablespace
```

Comment:
This way the DEFAULT_TABLESPACE has a higher priority than the model's Meta tablespace.

I think it's because of the DEFAULT_TABLESPACE [assignment](https://github.com/django/django/blob/main/django/db/models/options.py#L125) in options, but it might lead to unexpected behaviour if someone is not using the default.

Reply:
Thanks, @hartungstenio - I adjusted it.